ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -222,37 +222,58 @@ let detach_local_network_for_vm ~__context ~vm ~destination =
   * the pGPU is released and getting the PCI will fail.
   * *)
 let infer_vgpu_map ~__context ?remote vm =
+  let vf_device x = "vf:"^x in
```

Comment:
Perhaps this should be `vf_device_of` because `vf_device` is also used as a simple value later on.